### ISSUE TITLE
Your account has been suspended 

### ISSUE THREAD

**Issue by godka062** [Jimboom7/AlphaJong#76]
Is it related to using this program?
Have you checked other people's suspension besides me?

**Comment by godka062** [comment]
[](url
![정지](https://user-images.githubusercontent.com/104710977/176129864-6452199e-a61f-40b3-af4b-f9f9b8a93801.png)
)

**Comment by Jimboom7** [comment]
> Is it related to using this program?
Have you checked other people's suspension besides me?

Did you use other programs, too? You are the first who reports a ban, I've not heard of other cases yet.

**Comment by godka062** [comment]
Except for this program, only Korean patch programs were used on Japanese servers.
However, the Korean patch program was approved by Yosta.

**Comment by godka062** [comment]
Is there a problem with playtime?
I often use the program for more than 24 hours

**Comment by GALofAndroid** [comment]
![67C1BCBE-16F2-42AC-A39A-2221727A2C5F](https://user-images.githubusercontent.com/108333664/176145965-b5eb2e3b-0abd-4bc9-8f91-ec8b566a4e29.png)
I was banned too.

**Comment by pepethejelly** [comment]
I got banned today. mahjong soul team can find bot script user now. Your account will be banned as soon as you use it.

**Comment by 24802** [comment]
One thing I found out is that afk was detected in Chrome, but it is working normally in Firefox without detection notification.

**Comment by xgeesk** [comment]
they have gotten quite aggressive recently (both jp and en server). I only used the script in friendly rooms and non-ranked event rooms and they still got me

**Comment by legomeaker101** [comment]
I was banned to.
(last i use this service tbh... without a warning they just cut off a paying customer?)

**Comment by EBSMMF** [comment]
![无标题](https://user-images.githubusercontent.com/24271427/176441296-12a2cfbc-984d-457b-8cbf-c04ef185ef1b.png)
CN servers

**Comment by dioarame** [comment]
If other player used script with mahjong soul plus, It would be easily banned as I know : Third Party SW.
https://www.reddit.com/r/mahjongsoul/comments/ruvmbm/do_not_use_majsoul_plus/
And play 24/7 with scipt is also catched by them : suspicion of using a bot

IDK the script professionally, but I think AlphaJong is strong in terms of manipulation, not directly intervening in the game.
(Can the script be detected from the outside?)
In other words, it is thought that most of the stops are not caused by instant use, but by continuously playing the game all time/day.

**Comment by 312841943** [comment]
>请问你有用Firefox 24小时使用软件吗

**Comment by CaringCat** [comment]
> In other words, it is thought that most of the stops are not caused by instant use, but by continuously playing the game all time/day.

I have been running this script for the past few weeks on a separate account and Firefox instance, my account have not been banned yet. I only run it for a few hours a day.

**Comment by ZeroShira** [comment]
![Snipaste_2022-08-01_17-56-22](https://user-images.githubusercontent.com/92769606/182123794-f2b5f9a1-87d7-4c8c-8c94-596d786578a2.png)
cn server
用了3个号 2个ban
建议最近还是少用点

**Comment by WaterBeverage** [comment]
EN server, been testing 1.3.1 since available ~ about 2 months run time.
I've ran 666 4-player games (644 East & 22 South round) and 1,004 3-player games (928 East & 76 South round) and have not yet been banned.

However I did get a warning for AFK activity. Every time the Autostart portion refreshes the site, tampermonkey executes the script again. Around 155 script activation is when the bot enters the room and does nothing, triggering the Majsoul anti-afk penalty. After multiple violations, the game will ban the account.

![1](https://user-images.githubusercontent.com/111308533/190219453-e6659f35-b415-4210-962e-4aee8b541a73.png)

It takes a long time to reach that number of activation with the bot alone, but it can be achieved if you run multiple other scripts on the game's website. This may not be the main cause of bans, but it is what I've run into. 

I can confirm that it does indeed get to Master 1 on both 4-player and 3-player on default settings. It's slowly making its way into Saint.

**Comment by CruncyKukie** [comment]
![image](https://user-images.githubusercontent.com/93718509/194753441-63dfbbe3-0437-4c21-9c5b-766227337f1d.png)
got warned and banned for 15 days, phew my 200 usd didn't fly away

**Comment by ztmzzz** [comment]
被ban永久，只在活动的时期每天运行2小时

**Comment by wanxuntong** [comment]
> 

运气真好，我的被永久封禁

**Comment by Mizuhim3** [comment]
FYI, 
my accounts just got banned...
all my 8 accounts (good old one and new one) got banned using this script.
the MS Staff is so crue, even my main acc that only use the script only 3 times got banned too. Already got collab character and stuff.
I guess they detect my ip n it all checked accordingly...
My suggestion is, don't use this script for you own kind.

**Comment by liyishuai** [comment]
Banned permanently without warning, a few days after my last usage. Looks like they screen and ban in batches.
![94547F8D-32D4-47B5-B09D-84D9003F49FA](https://user-images.githubusercontent.com/7020805/212127940-714d04b2-2897-4aec-be04-4f5e736141a4.jpeg)
Not sure if they only warn users who have spent some money on it, as I haven’t spent a penny.

**Comment by legomeaker101** [comment]
> Banned permanently without warning, a few days after my last usage. Looks like they screen and ban in batches.
> ![94547F8D-32D4-47B5-B09D-84D9003F49FA](https://user-images.githubusercontent.com/7020805/212127940-714d04b2-2897-4aec-be04-4f5e736141a4.jpeg)
> Not sure if they only warn users who have spent some money on it, as I haven’t paid a penny.

I have spent money and I didn't get a warning

**Comment by Jimboom7** [comment]
I added a section to the readme that explains a method how to definitely avoid bans.
